docs/python_docs/python/tutorials/deploy/inference/image_classification_jetson.md

```diff
@@ -0,0 +1,130 @@
+
+## Installing MXNet v1.6 with Jetson support
+
+We start by installing MXNet dependencies
+```bash
+sudo apt-get update
+sudo apt-get install -y git build-essential libopenblas-dev libopencv-dev python3-pip
+sudo pip3 install -U pip
+```
+
+Then we download and install MXNet v1.6 wheel with Jetson support
+```bash
+wget https://mxnet-public.s3.us-east-2.amazonaws.com/install/jetson/1.6.0/mxnet_cu102-1.6.0-py2.py3-none-linux_aarch64.whl
+sudo pip3 install mxnet_cu102-1.6.0-py2.py3-none-linux_aarch64.whl
+```
+
+And we are done. You can test the installation now by importing mxnet from python3
+```bash
+>>> python3 -c 'import mxnet'
+```
+
+## Running a pre-trained ResNet-50 model on Jetson
+
+We are now ready to run a pre-trained model and run inference on a Jetson module. In this tutorial we are using ResNet-50 model trained on Imagenet dataset. We run the following classification script with either cpu/gpu context using python3.
+
+```python
+from mxnet.gluon import nn
+import mxnet as mx
+import numpy as np
+import urllib.request
+import cv2
+
+# set context
+ctx = mx.gpu()
+dtype = 'float32'
+bsize = 1
+
+# download model files
+path = 'http://data.mxnet.io/models/imagenet/'
+symbol,_ = urllib.request.urlretrieve(path+'resnet/50-layers/resnet-50-symbol.json')
+params,_ = urllib.request.urlretrieve(path+'resnet/50-layers/resnet-50-0000.params')
+label_file,_ = urllib.request.urlretrieve(path+'synset.txt')
+
+# load model
+input_names = ['data', 'softmax_label']
+net = nn.SymbolBlock.imports(symbol, input_names, params, ctx)
+net.cast(dtype)
+net.hybridize(static_alloc=True, static_shape=True)
+
+# load labels
+with open(label_file, 'r') as f:
+    labels = [l.rstrip() for l in f]
+
+# load image
+img_file,_ = urllib.request.urlretrieve('https://github.com/dmlc/web-data/blob/master/mxnet/doc/tutorials/python/predict_image/cat.jpg?raw=true')
+img = cv2.imread(img_file)
+img = cv2.cvtColor(img, cv2.COLOR_BGR2RGB)
+img = cv2.resize(img, (224, 224,))
+img = np.swapaxes(img, 0, 2)
+img = np.swapaxes(img, 1, 2)
+
+# format input
+batch = mx.nd.zeros((bsize,) + img.shape)
+for i in range(bsize):
+    batch[i] = img
+inputs = batch.astype(dtype)
+mx_img = [mx.nd.array(inputs,ctx), mx.nd.zeros((bsize,),ctx)]
+
+# infer
+results = net(*mx_img)
+prob = results[0].asnumpy()
+prob = np.squeeze(prob)
+a = np.argsort(prob)[::-1]
+for i in a[0:5]:
+    print('probability=%f, class=%s' %(prob[i], labels[i]))
+```
+
+After running the above script, you should get the following output showing the five classes that the image most relates to with probability:
+```bash
+probability=0.418679, class=n02119789 kit fox, Vulpes macrotis
+probability=0.293494, class=n02119022 red fox, Vulpes vulpes
+probability=0.029321, class=n02120505 grey fox, gray fox, Urocyon cinereoargenteus
+probability=0.026230, class=n02124075 Egyptian cat
+probability=0.022557, class=n02085620 Chihuahua
+```
```

Comment:
Accuracy validation is great, but can you add some code to show users how to do timing/performance-analysis too?

Reply:
That's a good idea. But I am not targeting performance analysis in this particular tutorial. Will put up another tutorial for performance after testing on more modules.